modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java

```diff
@@ -334,36 +309,31 @@ private static void printHelp() {
+        cli.print1("org.restlet.ext.platform.Introspector [credentials] [actions] [options] [--language swagger SWAGGER_DEFINITION_URL_OR_PATH | APPLICATION]");
 
         cli.print();
         cli.print0("DESCRIPTION");
         cli.print1(
-                "Publish to the APISpark platform the description of your Web API, represented by APPLICATION, the full name of your Restlet or JAX-RS application class or by the Swagger definition available at URL/PATH",
+                "Publish to the Restlet Cloud platform the description of your Web API, represented by APPLICATION, the full name of your Restlet or JAX-RS application class or by the Swagger definition available at URL/PATH",
                 "If the whole process is successful, it displays the url of the corresponding descriptor or connector cell.");
 
         cli.print();
         cli.print0("EXAMPLES");
         cli.print1(
-                "org.restlet.ext.apispark.Introspector -u 1234 -p Xy12 --create-descriptor com.acme.Application",
-                "org.restlet.ext.apispark.Introspector -u 1234 -p Xy12 --new-version --id 60 com.acme.Application",
-                "org.restlet.ext.apispark.Introspector -u 1234 -p Xy12 --update --update-strategy replace --id 60 --version 1 --language swagger http://acme.com/api/swagger");
+                "org.restlet.ext.platform.Introspector -u 1234 -p Xy12 --create-connector com.acme.Application",
+                "org.restlet.ext.platform.Introspector -u 1234 -p Xy12 --new-version --id 60 com.acme.Application",
+                "org.restlet.ext.platform.Introspector -u 1234 -p Xy12 --update --update-strategy replace --id 60 --version 1 --language swagger http://acme.com/api/swagger");
 
         cli.print();
         cli.print0("OPTIONS");
-        cli.print12("-h, --help" + "Prints this help.");
+        cli.print12("-h, --help", "Prints this help.");
         cli.print();
         cli.print1("[credentials]");
-        cli.print12("-u, --username username",
-                "The mandatory APISpark user name.");
-        cli.print12("-p, --password password",
-                "The mandatory APISpark user secret key.");
+        cli.print12("-u, --username username", "The mandatory Restlet Cloud user name.");
+        cli.print12("-p, --password password", "The mandatory Restlet Cloud user secret key.");
         cli.print();
         cli.print1("[actions]");
-        cli.print12("-d, --create-descriptor",
-                "Creates a new descriptor from introspection.");
-        cli.print12("-c, --create-connector",
-                "Creates a new connector from introspection.");
+        cli.print12("-c, --create-connector", "Creates a new connector from introspection.");
         cli.print12(
                 "-n, --new-version",
                 "Creates a new version of the descriptor/connector identified by the -i (--id) option");
```

Comment:
`Creates a new version of the connector identified by the`

Reply:
Bull's eye!